added possibility to set Range, to enable/disable Autocalibrate and t…

ChrSchultz opened the pull request:
…o calibrate maually ( see Datasheet v 1.2 : https://www.winsen-sensor.com/d/files/MH-Z14A.pdf)

Commit:
added possibility to set Range, to enable/disable Autocalibrate and to calibrate maually ( see Datasheet v 1.2 : https://www.winsen-sensor.com/d/files/MH-Z14A.pdf

tobiasschuerg commented on MHZ.cpp:
```diff
@@ -251,3 +254,41 @@ int MHZ::readCO2PWM() {
+  switch(range)
+  {
+    case 1:
+      _serial->write(cmd_2K,9);
```
```suggestion
      _serial->write(cmd_2K,9);
      break;
```

ChrSchultz committed:
Update MHZ.cpp

ChrSchultz committed:
Update MHZ.cpp

ChrSchultz committed:
Update MHZ.h

added calibrate Span, for complete functionality but commented out , due to professional use (req. constant atmosphere with 2k, 5k or 10k ppm CO2.).
corrected the bugs in MHZ.cpp also.

ChrSchultz commented:
made corrections and add the last feature for professional use,  but commented out.

ChrSchultz committed:
Update MHZ.cpp

Added comments for  Sensors with specific firmware.

tobiasschuerg reviewed (approved):
Thanks for the improvments!
